Formula/libglib.rb

```diff
@@ -0,0 +1,109 @@
+class Libglib < Formula
+  desc "Core application library for C"
+  homepage "https://developer.gnome.org/glib/"
+  url "https://download.gnome.org/sources/glib/2.72/glib-2.72.2.tar.xz"
+  sha256 "78d599a133dba7fe2036dfa8db8fb6131ab9642783fc9578b07a20995252d2de"
+  license "LGPL-2.1-or-later"
+
+  livecheck do
+    formula "glib"
+  end
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build
+  depends_on "gettext"
+  depends_on "pcre"
+
+  uses_from_macos "libffi"
+
+  on_linux do
+    depends_on "util-linux"
+  end
+
+  def install
+    # TODO: This is a workaround for `brew audit --new-formula`.
+    #       Use `patch` rather than `inreplace` (see also `glib`).
+    # replace several hardcoded paths with homebrew counterparts
+    inreplace "gio/xdgmime/xdgmime.c",
+              'xdg_data_dirs = "/usr/local/share/:/usr/share/";',
+              "xdg_data_dirs = \"#{HOMEBREW_PREFIX}/share/:/usr/share/\";"
+    inreplace "glib/gutils.c",
+              'data_dirs = "/usr/local/share/:/usr/share/";',
+              "data_dirs = \"#{HOMEBREW_PREFIX}/share/:/usr/share/\";"
+
+    # Disable dtrace; see https://trac.macports.org/ticket/30413
```

Comment:
It's been 11 years, did nobody report this upstream yet?

Reply:
I found an issue in upstream but it's still open.

https://gitlab.gnome.org/GNOME/glib/-/issues/653

Reply:
Can you replace the reference to that one then